.circleci/config.yml

```diff
@@ -261,6 +261,7 @@ workflows:
             branches:
               only: master
       - deploy-to-development:
+          context: api-nuget-token-context
```

Comment:
You need to add the context in deploy-to-staging and deploy-to-production jobs as well